fedimint-core/src/api.rs

```diff
@@ -613,6 +619,54 @@ impl WsClientConnectInfo {
     }
 }
 
+impl FromStr for WsClientConnectInfo {
+    type Err = anyhow::Error;
+
+    fn from_str(encoded: &str) -> Result<Self, Self::Err> {
+        let (hrp, data, variant) = bech32::decode(encoded)?;
+
+        ensure!(hrp == "fed", "Invalid HRP in bech32 encoding");
+        ensure!(variant == Bech32m, "Expected Bech32m encoding");
+
+        let bytes: Vec<u8> = Vec::<u8>::from_base32(&data)?;
```

Comment:
Can we add one bytes for "version", in case we ever need to change anything?

Reply:
Sure, I think the standard way is to put it into the HRP, so make it "fed1"

Reply:
:+1: That would be pretty.